EasyPost.sln

```diff
@@ -69,6 +51,14 @@ Global
 		{B05C7F02-568D-4587-91FF-D27B4DF5ED16}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{B05C7F02-568D-4587-91FF-D27B4DF5ED16}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{B05C7F02-568D-4587-91FF-D27B4DF5ED16}.Release|Any CPU.Build.0 = Release|Any CPU
+		{1B44D741-4EC2-4085-BF6E-8A81D330D77E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
What are all these and why do they seem to change with every PR?

Reply:
It's the Debug and Release configurations for the new .NET Standard project

Reply:
Ah, it's changed so much because we keep tweaking the platforms we'll use haha.

Reply:
Not necessarily, it's the project name. But yes, it's harmless.